portal/audit.py

```diff
@@ -31,7 +31,7 @@ def auditable_event(message, user_id, subject_id, context="other"):
     subject_id: The user id upon which the action was performed
 
     """
-    text = u"performed by {0} on {1}: {2}: {3}".format(
+    text = "performed by {0} on {1}: {2}: {3}".format(
```

Comment:
Won't this potentially create coercion errors if say the message contains unicode while we're still on py2?